Index: src/main/java/com/itheima/domain/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/itheima/domain/User.java	(date 1544775376299)
+++ src/main/java/com/itheima/domain/User.java	(date 1544775376299)
@@ -8,60 +8,60 @@
  * @date 2018/12/13 - 11:56
  */
 public class User implements Serializable {
-    private Integer uid;
-    private String username;
-    private String address;
-    private String sex;
-    private Date birthday;
+    private Integer userId;
+    private String userName;
+    private String userAddress;
+    private String userSex;
+    private Date userBirthday;
 
-    public Integer getUid() {
-        return uid;
+    public Integer getUserId() {
+        return userId;
     }
 
-    public void setUid(Integer uid) {
-        this.uid = uid;
+    public void setUserId(Integer userId) {
+        this.userId = userId;
     }
 
-    public String getUsername() {
-        return username;
+    public String getUserName() {
+        return userName;
     }
 
-    public void setUsername(String username) {
-        this.username = username;
+    public void setUserName(String userName) {
+        this.userName = userName;
     }
 
-    public String getAddress() {
-        return address;
+    public String getUserAddress() {
+        return userAddress;
     }
 
-    public void setAddress(String address) {
-        this.address = address;
+    public void setUserAddress(String userAddress) {
+        this.userAddress = userAddress;
     }
 
-    public String getSex() {
-        return sex;
+    public String getUserSex() {
+        return userSex;
     }
 
-    public void setSex(String sex) {
-        this.sex = sex;
+    public void setUserSex(String userSex) {
+        this.userSex = userSex;
     }
 
-    public Date getBirthday() {
-        return birthday;
+    public Date getUserBirthday() {
+        return userBirthday;
     }
 
-    public void setBirthday(Date birthday) {
-        this.birthday = birthday;
+    public void setUserBirthday(Date userBirthday) {
+        this.userBirthday = userBirthday;
     }
 
     @Override
     public String toString() {
         return "User{" +
-                "uid=" + uid +
-                ", username='" + username + '\'' +
-                ", address='" + address + '\'' +
-                ", sex='" + sex + '\'' +
-                ", birthday=" + birthday +
+                "uid=" + userId +
+                ", username='" + userName + '\'' +
+                ", address='" + userAddress + '\'' +
+                ", sex='" + userSex + '\'' +
+                ", birthday=" + userBirthday +
                 '}';
     }
 }
